src/pages/api/sendEmail.ts

```diff
@@ -0,0 +1,19 @@
+import { MailDataRequired } from "@sendgrid/helpers/classes/mail";
+import sgMail from "@sendgrid/mail";
+import { serverEnv } from "../../utils/env/server";
```

Comment:
Unless you have a good reason for this to go in ``pages/api`` move it to ``utils`` or we need to make another folder.

Reply:
api lives in api folder no? I'll remove it and put it in trpc once I check out the new branch

Reply:
Yeah I was just making sure you knew that the ``api`` folder was special in Next.